app/views/AdminView/components/Appendices/columnDefs.ts

```diff
@@ -0,0 +1,36 @@
+import { formatDate } from '@/utils/date';
+import { ColDef } from '@ag-grid-community/core';
+
+const columnDefs: ColDef[] = [
+  {
+    headerName: 'Name',
+    field: 'name',
+    hide: false,
+  },
+  {
+    headerName: 'Org',
+    field: 'organization',
+    hide: false,
+  },
+  {
+    headerName: 'Created',
+    valueGetter: ({ data }) => formatDate(data.createdAt),
```

Comment:
this is getting cut off, so I think it would be good to have a fixed width for this column